steps/http/steps.go

```diff
@@ -80,5 +84,8 @@ func (s Steps) InitializeSteps(ctx context.Context, scenCtx *godog.ScenarioConte
 	scenCtx.Step(`^the HTTP response body must be empty$`, func() error {
 		return session.ValidateResponseBodyEmpty(ctx)
 	})
+	scenCtx.Step(`^I store the field value of the json HTTP response "([^"]*)" in the context storage key "([^"]*)"$`, func(key string, ctxtKey string) error {
```

Comment:
To be consistent with common steps, I would rename the step text to:
`I store the element "([^"]*)" from the JSON HTTP response body in context "([^"]*)"$`

Note also that we are writing JSON instead of json. It's better to maintain the coherence.

Reply:
OK. Done